wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/settings/SettingsScreen.kt

```diff
@@ -38,7 +38,10 @@ import androidx.compose.ui.text.buildAnnotatedString
 import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.text.withStyle
 import androidx.compose.ui.unit.dp
+import com.android.identity.android.direct_access.DirectAccess
```

Comment:
Looks like there are a few imports here that are no longer needed. Please refresh the imports.